config/crd/bases/cloudsql.cloud.google.com_authproxyworkloads.yaml

```
@@ -994,6 +994,9 @@ spec:
                       privateIP:
                         description: PrivateIP (optional) Enable connection to the Cloud SQL instance's private ip for this instance. Default value is false.
                         type: boolean
+                      psc:
+                        description: PSC (optional) Enable connection to the Cloud SQL instance's private service connect endpoint for this instance. May not be used with PrivateIP. Default value is false.
```

i still find this read's weird. Do you? If you are saying `connection to the Cloud SQL instance's...` then I don't think you need to also have `for this instance`. It is repetitive and hinders the readability for me.
```suggestion
                        description: PSC (optional) Enable connection to the Cloud SQL instance's private service connect endpoint. May not be used with PrivateIP. Default value is false.
```

---

same in below places as well

---

Yeah, that's better. I've updated accordingly. 
Btw, these files are generated from the comments in authproxworkload_types.go by the Makefile using controller-runtime tools.